src/SearchInput/SearchInput.js

```diff
@@ -231,6 +241,8 @@ SearchInput.propTypes = {
             callback: PropTypes.func
         })
     ),
+    /** enable substring search */
+    subStringSearch: PropTypes.bool,
```

Comment:
What do you think about keeping the capitalization consistent with JavaScript's `substring` method?
`subStringSearch` -> `substringSearch`